### PR TITLE
feat(combined-report-ui): bump report package major version to 3

### PR DESCRIPTION
**Pull request by pownkel**
#### Description of changes

bump report package version from 2.2.1 to 3.0.0 (includes the new combined report UI and a breaking change to the interface for summary reports)

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Ran `yarn fastpass`
- [ ] Added/updated relevant unit test(s) (and ran `yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Review by JGibson2019 — APPROVED** [review]
approving so we're good to go once John's PR is merged

**Review by jalkire — APPROVED** [review]
PR is merged!

**Pull request merged**
